docs/docs/1.8.x/deployments/multi-zone.md

```diff
@@ -350,21 +350,25 @@ spec:
       status: online
 ```
 
-To consume the example service only within the same Kuma zone, you can rely on Kube-DNS and use the usual Kubernetes hostnames and ports, for example:
+The following are some examples of different ways to address `echo-server` in the
+`echo-example` `Namespace` in a multi-zone mesh.
+
+To send a request in the same zone, you can rely on Kubernetes DNS and use the usual Kubernetes hostnames and ports:
 
 ```sh
-<kuma-enabled-pod>$ curl http://echo-server:1010
+curl http://echo-server:1010
 ```
 
-To consume the example service across all zones in your Kuma deployment (that is, from endpoints ultimately connecting to the same global control plane), you can run:
+To send a request from any zone, you can [use the generated `kuma.io/service`](../explore/dpp-on-kubernetes.md#tag-generation) and [Kuma DNS](../networking/dns.md#dns):
```

Comment:
```suggestion
To send a request to any zone, you can [use the generated `kuma.io/service`](../explore/dpp-on-kubernetes.md#tag-generation) and [Kuma DNS](../networking/dns.md#dns):
```

Reply:
Might be worth saying that by default it will round robin between all zones and you can use localityAwareLB for more specific stuff